src/main/kotlin/com/terraformation/backend/customer/model/FunderUser.kt

```diff
@@ -17,16 +17,16 @@ import org.springframework.security.core.userdetails.UserDetails
 class FunderUser(
     val createdTime: Instant,
     override val userId: UserId,
-    override val authId: String?,
+    override val authId: String? = null,
     val email: String,
-    val emailNotificationsEnabled: Boolean,
-    val firstName: String?,
-    val lastName: String?,
-    val countryCode: String?,
-    val cookiesConsented: Boolean?,
-    val cookiesConsentedTime: Instant?,
-    override val locale: Locale?,
-    override val timeZone: ZoneId?,
+    val emailNotificationsEnabled: Boolean? = null,
```

Comment:
Does this need to be nullable, or could we default it to `true` instead since notifications are enabled for new users? That way we don't have to use null-safe operations every time we want to check it.

Reply:
Good thought, will update. I wasn't sure on the defaults for all of those since they're not specified in `IndividualUser`